src/components/datetime/DateTimeController.h

```diff
@@ -64,6 +64,7 @@ namespace Pinetime {
       const char* MonthShortToString();
       const char* DayOfWeekShortToString();
       static const char* MonthShortToStringLow(Months month);
+      const char* DayOfWeekShortToStringLow();
```

Comment:
just as in https://github.com/InfiniTimeOrg/InfiniTime/pull/1039 this function can be `const`

```suggestion
      const char* DayOfWeekShortToStringLow() const;
```